dlp/snippets/deid/deid_test.go

```diff
@@ -103,6 +103,25 @@ func TestDeidentifyDateShift(t *testing.T) {
 	}
 }
 
+func TestDeidentifyTableInfoTypes(t *testing.T) {
+	tc := testutil.SystemTest(t)
+
+	buf := new(bytes.Buffer)
```

Comment:
issue: don't use `new` here. Instead just declare `var buf bytes.Buffer`--this will automatically initialize the variable to its default/zero value.

Reply:
Resolved it, and considered it for all future samples.